epub33/core/index.html

```diff
@@ -5501,8 +5501,8 @@ <h4>Spread Placement</h4>
 						properties on the spine items for the two adjacent EPUB Content Documents, and omit the
 						properties on spine items where one-up or two-up presentation is equally acceptable.</p>
 
-					<p>EPUB Creators MAY only declare one <code>page-spread-*</code> property on any given spine
-						item.</p>
+					<p>EPUB Creators MUST NOT declare more than one <code>page-spread-*</code> property on any given
+						spine item.</p>
 
```

Comment:
The github interface does not allow me to comment on older lines but: respec reports missing links for the terms `page-spread-left` and `page-spread-right` properties in the description list at the beginning of §5.2.4. I suspect they should all be without `page-`.

Reply:
I think it may have been a problem with however you're viewing the document, as I'm not getting any errors. The links work fine for me. Was the import of the vocabulary maybe failing?

At any rate, I'm going to merge and see what happens, as the build should have failed if the links were broken.